site/src/main/resources/META-INF/hst-assembly/overrides/valueListManager.xml

```diff
@@ -8,6 +8,7 @@
                 <entry key="rights" value="/content/documents/administration/website/rights"/>
                 <entry key="lawfulbasis" value="/content/documents/administration/website/lawful-basis"/>
                 <entry key="creditbanner" value="/content/documents/administration/website/news-banner-type"/>
+                <entry key="highlighterlanguages" value="/content/documents/administration/website/highlighter-languages"/>
```

Comment:
Wonder if this is used anymore?  If not, can remove.

Reply:
You are correct, well spotted.